app/src/styles.sass

```diff
@@ -195,6 +195,7 @@ div.mat-menu-panel.nav-menu
   button.mat-menu-item:hover
     background-color: $nav-hover-bg
     color: white
+  margin-top: 4px
```

Comment:
Please move this line before line 189, such that all properties for the selector div.mat-menu-panel.nav-menu are grouped together.